code/modules/mob/living/simple_animal/hostile/mining_mobs/hivelord.dm

```diff
@@ -240,6 +240,8 @@
 				suit = pickweight(list(/obj/item/clothing/suit/hooded/explorer = 18, /obj/item/clothing/suit/hooded/cloak/goliath = 2))
```

Comment:
Exactly but you can order more via cargo, it seems to break balance of getting extra suits.